mscore/inspector/inspectorBase.cpp

```diff
@@ -344,7 +344,13 @@ void InspectorBase::valueChanged(int idx, bool reset)
 
       Score* score  = inspector->element()->score();
 
-      score->startCmd();
+      bool alreadyActive = false;
```

Comment:
The { } are not ok here. They should use banner style indenting. See https://musescore.org/en/handbook/musescore-coding-rules#Coding-style

But I'm not sure why this is required. I read your comment twice but I don't see it. Can you explain again differently why you need to check if the undostack is active here?